identity-apps-core/apps/authentication-portal/src/main/webapp/includes/branding-preferences.jsp

```diff
@@ -423,6 +424,7 @@
 
         if (brandingPreferenceResponse.has(PREFERENCE_KEY)) {
             brandingPreference = brandingPreferenceResponse.getJSONObject(PREFERENCE_KEY);
+            preferenceResourceType = brandingPreferenceResponse.getString(RESOURCE_TYPE);
```

Comment:
```suggestion
            preferredResourceType = brandingPreferenceResponse.getString(RESOURCE_TYPE);
```

Reply:
This is already used in multiple places. Shall we keep the same name here as well?

Reply:
This name is derived from `type` property from `brandingPreference` => brandingPreferenceType